lib/matplotlib/axes/_axes.py

```diff
@@ -3281,27 +3281,12 @@ def errorbar(self, x, y, yerr=None, xerr=None,
         kwargs = {k: v for k, v in kwargs.items() if v is not None}
         kwargs.setdefault('zorder', 2)
 
-        self._process_unit_info([("x", x), ("y", y)], kwargs, convert=False)
-
-        # Make sure all the args are iterable; use lists not arrays to preserve
-        # units.
-        if not np.iterable(x):
-            x = [x]
-
-        if not np.iterable(y):
-            y = [y]
-
+        x, y, xerr, yerr = self._process_unit_info(
+            [("x", x), ("y", y), ("x", xerr), ("y", yerr)], kwargs)
```

Comment:
So now `convert=True`, but I seem to be missing where the old conversion was that would be removed by this change?

Reply:
I think we just used to propagate unitful data to Line2D and rely on Line2D being (the only artist) able to handle them.